CONTRIBUTING.md

```diff
@@ -35,24 +35,127 @@ requests).
-
-## Pull Request Reviews
-
-All pull requests, even by members who have repository write access need to be
-reviewed and marked as "LGTM" before they will be merged.
\ No newline at end of file
+[AOSP Java Code Style](https://source.android.com/source/code-style). This code
+style is automatically enforced as part of the build, when you run the `check`
+task:
+
+```
+./gradlew check
+```
+Pull requests are expected to at least meet the following criteria:
+
+- The build of the `check` task must succeed. If it does not, reviewers will
+  simply reply to the PR with "Please fix the build errors", and review no
+  further until this criteria is met.
+
+- The initial submission should consist of a single commit. Review comments
+  should be addressed through additional commits on the same branch. Avoid
+  rebasing or history rewriting in your PR where possible, to preserve the
+  review history. Intermediary commits will all be discarded when the PR 
+  is accepted, as the reviewer will "squash and merge" your PR once ready.
+  
+- It is the submitters responsibility to ensure that a PR can be cleanly
+  merged into the master branch.
+
+- All code submitted must have at least 60% coverage for the affected lines.
+  Reviewers may request higher coverage, or additional tests generally, if 
+  the changes made warrant this. Generally, aim for a net increase in code
+  coverage for the project with each PR.
+  
+- Code submitted must not drop the overall coverage of the project below 80%.
+  If the code coverage is already below 80%, then contributions must increase
+  the net coverage to be accepted.
+  
+- Code changes that have implications for the specification must have an 
+  associated thread of discussion here on [list][oidf-ac-list], and consensus 
+  must have been reached that the change is worthwhile. Preferably, a separate
+  pull request should already have been merged into to the 
+  [OpenYOLO-Spec repository][spec-repo] before a code change is submitted.
+  
+- Code changes that affect the API must also be demonstrated as part of the 
+  test and sample applications.
+
+- Code changes that affect the SPI must also be demonstrated as part of the 
+  test and demo providers.
+
+- Complex changes _should_ be broken into multiple smaller pull requests for 
+  review, and these should not be directly submitted to the master branch. 
+  Instead, they must be submitted to a feature branch until the complete 
+  set of changes is made, and then merged into master upon agreement that
+  the work is complete.
+
+- As a general guideline, pull requests should not contain more than a 1000
+  line delta. Reviewers may still agree to review PRs that are larger than
+  this at their discretion, or they can request that the PR be split into
+  smaller changes for more effective review.
+
+## Responsibilities of reviewers
+
+Reviewers (members of the 
+[@openid/openyolo-android-maintainers][maintainers-team] team) are expected
```

Comment:
FYI, link is broken atm.

Reply:
Removed, looks like it's not visible to anyone outside the OIDF org.